internal/templates/assisted-installer/template.go

```diff
@@ -119,68 +165,57 @@ spec:
 {{ end }}
   pullSecretRef:
     name: "{{ .Spec.PullSecretRef.Name }}"
+{{ if .SpecialVars.CurrentNode.IgnitionConfigOverride }}
+  ignitionConfigOverride: '{{ .SpecialVars.CurrentNode.IgnitionConfigOverride }}'
```

Comment:
No, this is a different thing.

IIRC the ignition override here is meant for the host install (pointer) ignition overrride. These are things that should be present on the node after it reboots into the installed disk.

The infraenv ignition overrides are specifically for discovery.

Reply:
Removed the `ignitionConfigOverride` part as we can create a separate issue for that when an actual use-case arises.